Formula/nettle.rb

```diff
@@ -26,9 +26,13 @@ def install
                                           "get_pc_thunk|(_*chkstk_darwin)"
     end
 
+    args = []
+    args << "--build=aarch64-apple-darwin#{`uname -r`.to_i}" if Hardware::CPU.arm?
```

Comment:
You don't need to call `to_i`; it works with the full number, and the `build` value typically includes the entire kernel version.
```suggestion
    args << "--build=aarch64-apple-darwin#{`uname -r`.chomp}" if Hardware::CPU.arm?
```